RogueMaster/flipperzero-firmware-wPlugins
# Update to FlipBIP v0.0.7, QR Code addrs, better UI perf

# What's new

- QR Code receive addresses
- Faster UI performance

# Verification 

- Verify QR files corresponding to receive addresses are created when selecting `View X Wallet` in `apps_data/flipbip/*.qrcode`
- Verify these files can be opened with the `qrcode` app
- Verify changing pages to scroll past addresses in the `View X Wallet` has no pauses

# Checklist (For Reviewer)
